### DISCUSSION TITLE
Need help with -X settings, or other settings to make signal decoding more reliable

### DISCUSSION THREAD

**Discussion by jz1212** [merbanan/rtl_433#2694] [Q&A]
Hello, I'm new to SDR, and been trying to find a way to troubleshoot Teensy+RFM69 project I had been working on
I can get Universal Radio Hacker to more or less reliably receive and decode the messages with some manual editing, but when I try to set up RTL_433 to do the same thing I run into issues, very unreliable signal decoding and I just don't know what setting to tweak to make it work reliably
I'm attaching screenshots of URH with some explanations on them, and screenshots of RTL_433 running on ubuntu (command lines to get the result are at the bottom of the screenshots)
![Screenshot 2023-10-25 143315](https://github.com/merbanan/rtl_433/assets/126067949/8a4551be-e3fb-4fd8-9dac-e85e60298aa4)
![Screenshot 2023-10-25 143406](https://github.com/merbanan/rtl_433/assets/126067949/9eba0185-2fb2-436c-978d-8f9cc1b4548a)
![Screenshot 2023-10-25 143431](https://github.com/merbanan/rtl_433/assets/126067949/a38e60a5-b8fc-4ae6-ad4f-7fa69537daf0)
![Screenshot 2023-10-25 143758](https://github.com/merbanan/rtl_433/assets/126067949/4d17d407-f3f4-49ea-aaef-f8ecde542935)
![Screenshot 2023-10-25 144244](https://github.com/merbanan/rtl_433/assets/126067949/d6169b42-878e-4b5e-8d43-df8279b6511b)
![Screenshot 2023-10-25 144438](https://github.com/merbanan/rtl_433/assets/126067949/0f671b18-20fc-4e56-957b-ffdd04d7d6a3)

Here are the URH files for reference:
[URH_recordings.zip](https://github.com/merbanan/rtl_433/files/13165764/URH_recordings.zip)

**Comment by zuckschwerdt** [comment]
Try to use a more current version of rtl_433, e.g. `-a` is not available anymore.
`5555555393906...` in the last example is `aaaaa72720...` so the match you show should have worked?

I would not use a reset of 1000. The maximum sane value here is maybe 170.

**Comment by jz1212** [comment]
rtl_433 version is what got installed in ubuntu using the commands from the install guide, shouldn't it be the newest version?
-a was probably remnant of different options I tried to use from googling trying to find info on how to decode unknown signals

and the match 0x7272 I used did return the messages, but I can't figure out why it is showing 0000000 instead of the bytes after the 5553939 most of the time, so I'm guessing there is a decoding issue, noise maybe? In URH at the top left there is a Noise variable, set to 0,01, that helps to make the signal look presentable, without it it would very noisy, perhaps that is what is confusing rtl_433, but I can't figure out which options would make it apply filtering to reduce noise
I later did apply an option to save received signals in a file, then opened that file in URH, and indeed it looks very noisy, but if I clean it up using the Noise option and apply low pass filter, then it cleans up nicely and I can decode it just like shown in the URH screenshots

reset option value was from the help example line, I'm not aware of what it actually does, I'll try the suggested 170 when I get back to that setup in the evening

btw RTL dongle I use is this one https://www.amazon.de/-/en/dp/B009U7WZCA?psc=1&ref=ppx_yo2ov_dt_b_product_details

**Reply by zuckschwerdt** [reply]
Try using ` -Y autolevel -M level -M noise` and `-Y classic` to get a better demod maybe. But you might need to install a current version by compiling yourself.
